redfish-core/lib/log_services.hpp

```diff
@@ -1147,6 +1169,46 @@ inline void requestRoutesSystemLogServiceCollection(App& app)
                     "/xyz/openbmc_project/object_mapper",
                     "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths", "/",
                     0, std::array<const char*, 1>{postCodeIface});
+
+                // Make sure the "HardwareIsolation" service support in the
+                // backend, if it is available then, add into "LogServices".
+                crow::connections::systemBus->async_method_call(
+                    [asyncResp](const boost::system::error_code ec,
+                                const std::vector<std::string>& subtreePath) {
+                        if (ec)
+                        {
+                            BMCWEB_LOG_ERROR
+                                << "D-Bus response error [" << ec.value()
+                                << " : " << ec.message() << "] "
+                                << "when tried to check the HardwareIsolation "
```

Comment:
return http error?

Reply:
This code removed as part of compiler flag changes :)